src/Poseidon/SecondaryTypes.hs

```diff
@@ -33,67 +48,62 @@ data IndividualInfo = IndividualInfo
     { indInfoName    :: String
     , indInfoGroups  :: [String]
     , indInfoPacName :: String
-    } deriving (Show, Ord)
+    } deriving (Show, Ord, Generic)
 
 instance Eq IndividualInfo where
     (==) (IndividualInfo a1 b1 c1) (IndividualInfo a2 b2 c2) = a1 == a2 && head b1 == head b2 && c1 == c2
 
 instance ToJSON IndividualInfo where
-    toJSON x = object [
-        "name"    .= indInfoName x,
-        "group"   .= indInfoGroups x,
-        "pacName" .= indInfoPacName x]
+    toEncoding = genericToEncoding defaultOptions
```

Comment:
Why do you want to go back to a generic instance definition here?

Reply:
I do not have a good answer here. I guess I liked the simplicity of just having the compiler set all these up for us, since the user will basically never see the JSON anyway. But happy to define manual instances... doesn't really matter

Reply:
My thought process is that  a custom instance makes it easier to track a change in the API. We don't want to accidentally modify it (e.g. by changing some type), but want to be conscious about these changes, right?

Reply:
Ah yes, good point. Well, OK then, but I suppose this applies to almost all JSON instances then, right? Or, at least those involved in the Server-Client communication?

Reply:
OK, I've now hand-written all JSON instances for Server-Client involved types.